File: ChX3DTouchDemo/ChX3DTouchDemo/ViewController.m
//
//  ViewController.m
//  ChX3DTouchDemo
//
//  Created by Xu Chen on 2018/8/11.
//  Copyright © 2018年 xu.yzl. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
}

- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
    UIViewController *nextVC = [[NSClassFromString(@"TableViewController_A") alloc] init];
    [self presentViewController:nextVC animated:YES completion:nil];
}

@end

File: ChX3DTouch(Swift4.0)/ChX3DTouch(Swift4.0)/ViewController.swift
//
//  ViewController.swift
//  ChX3DTouch(Swift4.0)
//
//  Created by Xu Chen on 2018/8/18.
//  Copyright © 2018年 xu.yzl. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        show(TableViewController_A(), sender: self)
    }


}

